lib/gollum-lib/icon.rb

```diff
@@ -0,0 +1,33 @@
+module Gollum
+  class Icon
+    # See if octicons gem is available
+    @@octicons = false
+    begin
+      require 'octicons'
+      @@octicons = true
+    rescue LoadError; end
+    
+    def self.get_icon(name, options={}, for_css=false)
+      defaults = {height: 16, width: 16, class: ''}
+      options = defaults.merge(options)
+      if for_css
+        options.delete(:height)
+        options.delete(:width)
+      end
+      if @@octicons && octicon = ::Octicons::Octicon.new(name, options)
+        octicon.to_svg
+      else
+        height = %Q(height="#{options[:height]}") if options[:height]
```

Comment:
The implementation of this class looks good to me! I would suggest abstracting this else block into a separate method, `render_default_icon(name, options)` or some such.

Reply:
Done. 👍